tests/test_dm_control.py

```diff
@@ -22,10 +21,7 @@ def run_task(domain_name, task_name):
     time_step = dm_control_env.reset()
     action_spec = dm_control_env.action_space
     for _ in range(5):
-        if 'CI' in os.environ:
-            print("Skipping rendering test")
-        else:
-            dm_control_env.render()
+        dm_control_env.render()
```

Comment:
Why deleting the CI condition?

Reply:
We can now render on the CI using a fake X display inside Docker